src/commands/install-deps.yml

```diff
@@ -26,8 +26,9 @@ steps:
       steps:
         - restore_cache:
             keys:
-              - << parameters.key >>-{{ checksum "./Gemfile.lock"  }}-{{ .Branch }}
-              - << parameters.key >>-{{ checksum "./Gemfile.lock"  }}
+              - << parameters.key >>
+              - gems-v1-{{ checksum "./Gemfile.lock"  }}-{{ .Branch }}
```

Comment:
We wouldnt want to hard code these back up keys that are locked to v1

Reply:
In that case, should we hardcode to lock to `gems-` and call it a day? Or just not have a fallback?

Reply:
What about something like:

```yml
keys:
  - << parameters.key >>-{{ checksum << parameters.lockfile >>  }}-{{ .Branch }}
  - << parameters.key >>-{{ checksum << parameters.lockfile >>  }}
  - << parameters.key >>
```

I think that solves for both issues, while still not requiring a whole bunch of extra complexity. That is, assuming we can using a `<<` template inside of a `{{` template. It doesn't seem like this is something that is likely to change a whole lot in the future, once a simple balance is reached.

Reply:
I'm also wondering why we need `.Branch` at all. Could we not instead use:

```yml
keys:
  - << parameters.key >>-{{ checksum << parameters.lockfile >>  }}
  - << parameters.key >>
```

I guess it depends on how likely `checksum` is to have collisions, but if that's pretty low then we wouldn't need to generate a new cache for every branch.

Reply:
Side note, [another comment](https://github.com/CircleCI-Public/ruby-orb/issues/41#issuecomment-687464802) suggested adding `{{ arch }}` which might be a good idea to accommodate cross-platform development.